packages/website/components/GlobalStyles.tsx

```diff
@@ -48,6 +48,20 @@ export const GlobalStyles = () => {
           margin: 0;
         }
 
+        p,
+        a,
+        li {
+          font-family: ${tokens.fontStackPrimary};
+          font-size: ${tokens.fontSizeL} !important;
+          line-height: ${tokens.lineHeightL} !important;
+        }
+
+        h2 {
+          font-family: ${tokens.fontStackPrimary};
+          font-size: ${tokens.fontSizeL} !important;
+          line-height: ${tokens.lineHeightL} !important;
+        }
```

Comment:
We would probably need to figure out how to overwrite other typography styles accordingly